docs/source/basic/library.rst

```diff
@@ -211,7 +211,7 @@ Memory Management
 
 The memory allocation function of the *alpaka* library (``alpaka::allocBuf<TElem>(device, extents)``) is uniform for all devices, even for the host device.
 It does not return raw pointers but reference counted memory buffer objects that remove the necessity for manual freeing and the possibility of memory leaks.
-Additionally the memory buffer objects know their extents, their pitches as well as the device they reside on.
+Additionally the memory buffer objects know their extents, their pitches (space allocated in including the padding) as well as the device they reside on.
```

Comment:
This is not correct. The pitches are strides. They are the number of elements to skip to reach the next element/row/slice, etc.
The mental model for accessing a multidimensional buffer/array is: `*(ptr + dot_product(index, pitches))`.

Reply:
Yes, with data+padding it makes a stride. I should have written explicitly.

Reply:
"due to padding, allocated number of bytes may be more than the data, pitch value gives the correct stride for row-major access"